app/views/IncomeSourceSummary.scala.html

```diff
@@ -76,7 +76,7 @@
     title = title,
     authedUser = Some(user),
     pageHeader = Some(header),
-    articleClasses = None,
+    articleClasses = Some("incomeSourceSummary"),
```

Comment:
Do you still need this class?

Reply:
I won't if Rachel approves the design that only requires class update

Reply:
updated as per suggestion